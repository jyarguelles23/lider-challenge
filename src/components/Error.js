import React from "react";
import image from "../assets/error.PNG";

const Error = () => (
  <div className="col-12 col-sm-6 col-md-4 col-lg-3 mb-4">
    <img src={image} alt="Error" />
  </div>
);

export default Error;
