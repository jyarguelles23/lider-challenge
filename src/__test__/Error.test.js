import Error from "../components/Error";
import React from "react";
import renderer, { ReactTestRenderer } from "react-test-renderer";

let tree = ReactTestRenderer;

describe("<Error/>", () => {
  beforeEach(() => {
    tree = renderer.create(<Error />);
  });

  it("should render without crashing", () => {
    expect(tree.toJSON()).toMatchSnapshot();
  });

  it("should render an error image", () => {
    tree = renderer.create(<Error />);
    const error = tree.root;
    expect(
      error.findByType("img").props.alt
    ).toBe("Error");
  });
});
