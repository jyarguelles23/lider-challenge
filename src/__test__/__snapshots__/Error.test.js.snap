// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error/> should render without crashing 1`] = `
<div
  className="col-12 col-sm-6 col-md-4 col-lg-3 mb-4"
>
  <img
    alt="Error"
    src="error.PNG"
  />
</div>
`;
